src/Altinn.App.Api/Altinn.App.Api.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <OutputType>Library</OutputType>
-    <Version>5.1.0-alpha.2</Version>
-    <AssemblyVersion>5.1.0.0</AssemblyVersion>
+    <Version>5.2.0</Version>
```

Comment:
Btw, can you change the version to 5.2.0-alpha.1
We publish the nuget package with an alpha until it tested and verified from our side.

Reply:
I guess this also goes for `Altinn.App.PlatformService`?

Reply:
Yep, PlatformService, Common and Api.